docker-compose.yml

```diff
@@ -1,9 +1,10 @@
 version: '2'
 services:
   postgres:
-    image: timms/postgres-logging:11.5
+    image: timms/postgres-logging:15.0
```

Comment:
Let's also check 16.1. I think you can just use `postgres:16.1-alpine`

Reply:
Times didnt have pne available i think but i will snap it for something else